tempesta_fw/sock_clnt.c

```diff
@@ -45,28 +45,62 @@
  * ------------------------------------------------------------------------
  */
 
+/**
+ * TfwConnection extension for client sockets.
+ *
+ * @conn		- The base structure. Must be the first member.
+ * @keepalive_timer	- The keep-alive timer for the connection.
+ * @timeout		- The keep-alive timeout for the connection.
+ *
+ */
+typedef struct {
+	TfwConnection		conn;
+	struct timer_list	keepalive_timer;
```

Comment:
It would change this to `ka_timer` as the name appears to be very long. It's purely cosmetic though, so please feel free to keep your own name.